### ISSUE TITLE
fasterq-dump does not produce expected files

### ISSUE THREAD

**Issue by jacobHog** [ncbi/sra-tools#861]
Hello,

I have downloaded the sra-toolkit from Anaconda (https://anaconda.org/bioconda/sra-tools) and downloaded an .sra file using the command: `prefetch SRR20073591`.
The .sra file is located here: /faststorage/project/Biof/testdir/SRR20073591/SRR20073591.sra
When I navigate to the directory and use this command: `fasterq-dump SRR20073591.sra`, I get an output file called SRR20073591.fastq. However, I was expecting to get separate R1 and R2 sequencing files as well.

Would anyone be kind enough to assist me with this issue?

**Comment by durbrow** [comment]
That [run](https://trace.ncbi.nlm.nih.gov/Traces/?view=run_browser&page_size=10&acc=SRR20073591&display=metadata) contains 3 reads, reads 1 and 2 are technical reads, which fasterq-dump skips by default (try adding `--include-technical` to the command). If you have reason to think this read layout is incorrect, please email the curators at sra@ncbi.nlm.nih.gov and let them know.

**Comment by jacobHog** [comment]
Thank you for the prompt response and for maintaining the the SRA Toolkit!
`--include-technical` did indeed create 3 files when using fasterq-dump, however I noticed that the file naming differs from what is described in the guide (https://github.com/ncbi/sra-tools/wiki/HowTo:-fasterq-dump). 
The files I recieved were named SRRX_1.fastq, SRRX_2.fastq, and SRRX_3.fastq.
As a user reading the guide I get the interpretation that I should use the _1.fastq and _2.fastq file, however the files that I downloaded I see that the _2.fastq file contains the UMI + cell barcodes and _3.fastq file contains the cDNA read. (I got guidance from biostars users).
